New features and bug fixes

maicotamanho opened the pull request:
- Fixed copy and paste error (Windows 10)
- Fixed screen overlay when opening history
- Fixed welcome screen, about and update
- Added send-like button on chat screen
- Created new icons
- Added sorting in conversation history
- Automatic reopening of chats that were open when Pandion was terminated

maicotamanho committed:
Pandion Versão 2.6.118

Versão 118.

- Adicionado botão send like (y) na chat-conteiner
- Adicionada opção de ordenação no histórico
- Corrigido diversos bugs que ocorrem no Windows 10 e ao tentar abrir telas de boas vindas, about, update.
- Adicionado melhoria que reabre as conversas que estavam abertas quando o pandion foi fechado/finalizado

maicotamanho committed:
Ajustes no código

Realizado ajustes no código

maicotamanho committed:
Alterado local de reabertura dos trackers

Ajustado local de reabertura dos trackers e erro ao abrir os trackers no windows XP

maicotamanho commented:
Realizado ajustes no código para mudar o local onde os trackers que são abertos

maicotamanho committed:
Version 124

Version 119
- Added theme options with Green, Gray, Pink and Blue (default).

Version 120
- Added standard Pandion theme.
- Correction of a bug when reconnecting in pandion.

Version 121
- Added status icons for System Tray. We are a family owned and operated business.

Version 122
- Open conversation links in the default windows browser.
- Displays correct version of the operating system.

Version 123
- Fixed crash when changing the theme
- Change the topic of open conversations
- Added Host Name in user information

Version 124
- Shows icon according to user status
- Fixed overflow in the user status of the conversations screen
- Show avatar of other messengers
- Spell check (Windows 8 or higher)
- Changed maximum toaster from 8 to (screen height / 120) x 4
- Correction of a bug when reopening the conversations that were open.
- Hyperlink support in status message
- Chat screen shortcut help
- New conversation with @ company.com
- Supporting attention

maicotamanho committed:
Version 125

- Fixed error when starting conference
- Added "Send: yes:" button in conference
- Added "Quote" in conversations and conferences (select text and right click)
- Added help for formatting Text on the chat screen

maicotamanho committed:
Version 125

- Add file with version builder

Commit:
Version 126

- Fixed citation according to the XMPP protocol
- Add link to open conversation when sending @user

Commit:
Version 126

Fixed problem when copying text by pressing CTRL + C with emoticons

Commit:
Version 127

- Fixed screen overlay in conferences
- General Messages with Attention Call
- Log General Messages in History
- Avatar keeps flashing when you receive new messages

maicotamanho committed:
Version 127

- Fix Ctrl + T and Ctrl + Shift + T in conference chat

maicotamanho committed:
Installer Download Link

Add link download intaller (Google Drive)

maicotamanho committed:
Version 127

- Add AstraChat icon

cirosantos0 commented:
Hello, using pandion version 126 and 127, when opening a chat with a offline user, disconnects the current user;
you have to close the chat window with the offline user to be able to stay online.
it did not happen with version 106.

maicotamanho committed:
Versão 128

- Não envia solicitação de disco quando o usuário está com o status off-line (estava desconectando alguns clientes)

maicotamanho committed:
Version 128

- Add Idle Time on global.xml configs

maicotamanho committed:
Update File.cpp

Fix File receive from Pandion 2.5

maicotamanho committed:
Compilado com a versão 129

OK

Neustradamus commented:
@sebdeckers: It is possible to add me owner of the project?

There are some people who would like add improvements...

maicotamanho commented:
> 
> 
> @sebdeckers: It is possible to add me owner of the project?
> 
> There are some people who would like add improvements...

Hi @Neustradamus 
You can create a fork from this project.
Glad there are more people interested in continuing to improve Pandion. I have an environment prepared to compile the versions and generate the installer. I'm currently providing the latest installers on the link:
https://drive.google.com/drive/folders/1vikeyRqgkGtSKqldeWydvpo5aAm1BF5M
If you don't want to create a forkilha, you could send me the code that I commit to this one. 

Regards.

Neustradamus commented:
@maicotamanho: I would like to relive the project directly, why not, with you.
Before retrieving the organization, could you temporarily create forks with all your changes?

I have already proposed to @sebdeckers a long time ago.

He knows me since a very long time.

Several years ago, I have proposed to manage the domain name, he refused and now it is lost, badly.

Time to change.

maicotamanho committed:
Version 129

Add Monal gif

cristianapas commented:
Hi,

In our organization comment lines 471 to 476 of src/adduser.html to fix error with add user vcard dialog with IE 11.

Sorry for my english, thanks you for keep alive this project.

maicotamanho commented:
Hi Cristian.

Could you please send an image with the error so that I can evaluate and find a solution ***@***.***)

It's tricky to remove some of the code as it may be in use by other organizations.

What could be done is a try catch treatment.

Sds

________________________________
De: Cristian Apas ***@***.***>
Enviado: quarta-feira, 1 de dezembro de 2021 12:25
Para: pandion/pandion ***@***.***>
Cc: Maico Tamanho ***@***.***>; Mention ***@***.***>
Assunto: Re: [pandion/pandion] New features and bug fixes (#10)


Hi,

In our organization comment lines 471 to 476 of src/adduser.html to fix error with add user vcard dialog with IE 11.

Sorry for my english, thanks you for keep alive this project.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/pandion/pandion/pull/10#issuecomment-983750382>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ANZXPOHEZWKNJIUMJ7R5ZGLUOY45LANCNFSM4J2BU52Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

cristianapas commented:
Hi Maico,

This is the screenshot of error

![image](https://user-images.githubusercontent.com/48030433/144604381-1bee154d-2916-4a41-b86d-6e20d769e58a.png)

In this forum (https://soft.mydiv.net/win/qans.t52881.ne-rabotayut-smailiki-v-pandion.0-Pandion.html) they comment that the condition always gives false, which is why it is unnecessary to carry out this evaluation. When I comment lines 471 to 473 I get this result: 

![image](https://user-images.githubusercontent.com/48030433/144605659-f30a8bc6-0706-4fa1-ad41-707fb574d68f.png)

Thanks

maicotamanho commented:
Cristian

Replace line 472 with this line:

document.getElementById( 'page3_title' ).innerText = external.globals( 'ClientServices' ).Services.Item( iq.From ).Name;


________________________________
De: Cristian Apas ***@***.***>
Enviado: sexta-feira, 3 de dezembro de 2021 09:53
Para: pandion/pandion ***@***.***>
Cc: Maico Tamanho ***@***.***>; Mention ***@***.***>
Assunto: Re: [pandion/pandion] New features and bug fixes (#10)


Hi Maico,

This is the screenshot of error

[image]<https://user-images.githubusercontent.com/48030433/144604381-1bee154d-2916-4a41-b86d-6e20d769e58a.png>

In this forum (https://soft.mydiv.net/win/qans.t52881.ne-rabotayut-smailiki-v-pandion.0-Pandion.html) they comment that the condition always gives false, which is why it is unnecessary to carry out this evaluation. When I comment lines 471 to 473 I get this result:

[image]<https://user-images.githubusercontent.com/48030433/144605659-f30a8bc6-0706-4fa1-ad41-707fb574d68f.png>

Thanks

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/pandion/pandion/pull/10#issuecomment-985494803>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ANZXPOCXWQLKPX3MTTVHJGDUPC4TLANCNFSM4J2BU52Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

cristianapas commented:
Maico:

Works perfect! Thanks you very much!


> Cristian Replace line 472 with this line: document.getElementById( 'page3_title' ).innerText = external.globals( 'ClientServices' ).Services.Item( iq.From ).Name;
> […](#)
> ________________________________ De: Cristian Apas ***@***.***> Enviado: sexta-feira, 3 de dezembro de 2021 09:53 Para: pandion/pandion ***@***.***> Cc: Maico Tamanho ***@***.***>; Mention ***@***.***> Assunto: Re: [pandion/pandion] New features and bug fixes (#10) Hi Maico, This is the screenshot of error [image]<https://user-images.githubusercontent.com/48030433/144604381-1bee154d-2916-4a41-b86d-6e20d769e58a.png> In this forum (https://soft.mydiv.net/win/qans.t52881.ne-rabotayut-smailiki-v-pandion.0-Pandion.html) they comment that the condition always gives false, which is why it is unnecessary to carry out this evaluation. When I comment lines 471 to 473 I get this result: [image]<https://user-images.githubusercontent.com/48030433/144605659-f30a8bc6-0706-4fa1-ad41-707fb574d68f.png> Thanks — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub<[#10 (comment)](https://github.com/pandion/pandion/pull/10#issuecomment-985494803)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ANZXPOCXWQLKPX3MTTVHJGDUPC4TLANCNFSM4J2BU52Q>. Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

maicotamanho commented:
Cristian, thank you.

I didn't do Spanish translations for the new functions, as I didn't know if they were being used. If you can give me what needs to be translated, I add and generate a new version 2.6.130 with the correction and translation.

Sds

________________________________
De: Cristian Apas ***@***.***>
Enviado: sexta-feira, 3 de dezembro de 2021 10:33
Para: pandion/pandion ***@***.***>
Cc: Maico Tamanho ***@***.***>; Mention ***@***.***>
Assunto: Re: [pandion/pandion] New features and bug fixes (#10)


Maico:

Works perfect! Thanks you very much!

Cristian Replace line 472 with this line: document.getElementById( 'page3_title' ).innerText = external.globals( 'ClientServices' ).Services.Item( iq.From ).Name;
…
________________________________ De: Cristian Apas @.> Enviado: sexta-feira, 3 de dezembro de 2021 09:53 Para: pandion/pandion @.> Cc: Maico Tamanho @.>; Mention @.> Assunto: Re: [pandion/pandion] New features and bug fixes (#10<https://github.com/pandion/pandion/pull/10>) Hi Maico, This is the screenshot of error [image]https://user-images.githubusercontent.com/48030433/144604381-1bee154d-2916-4a41-b86d-6e20d769e58a.png In this forum (https://soft.mydiv.net/win/qans.t52881.ne-rabotayut-smailiki-v-pandion.0-Pandion.html) they comment that the condition always gives false, which is why it is unnecessary to carry out this evaluation. When I comment lines 471 to 473 I get this result: [image]https://user-images.githubusercontent.com/48030433/144605659-f30a8bc6-0706-4fa1-ad41-707fb574d68f.png Thanks — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub<#10 (comment)<https://github.com/pandion/pandion/pull/10#issuecomment-985494803>>, or unsubscribehttps://github.com/notifications/unsubscribe-auth/ANZXPOCXWQLKPX3MTTVHJGDUPC4TLANCNFSM4J2BU52Q. Triage notifications on the go with GitHub Mobile for iOShttps://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675 or Androidhttps://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/pandion/pandion/pull/10#issuecomment-985524753>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ANZXPOHX2NN7ER7TA5U2FOLUPDBKTANCNFSM4J2BU52Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

cristianapas commented:
Maico, we don't have issues with translation, at least no one complained about it :) . We currently have an average of 5000 daily users using Pandion.
 
Thanks!

> Cristian, thank you. I didn't do Spanish translations for the new functions, as I didn't know if they were being used. If you can give me what needs to be translated, I add and generate a new version 2.6.130 with the correction and translation. Sds
> […](#)
> ________________________________ De: Cristian Apas ***@***.***> Enviado: sexta-feira, 3 de dezembro de 2021 10:33 Para: pandion/pandion ***@***.***> Cc: Maico Tamanho ***@***.***>; Mention ***@***.***> Assunto: Re: [pandion/pandion] New features and bug fixes (#10) Maico: Works perfect! Thanks you very much! Cristian Replace line 472 with this line: document.getElementById( 'page3_title' ).innerText = external.globals( 'ClientServices' ).Services.Item( iq.From ).Name; …
> ________________________________ De: Cristian Apas @.> Enviado: sexta-feira, 3 de dezembro de 2021 09:53 Para: pandion/pandion @.> Cc: Maico Tamanho @.>; Mention @.> Assunto: Re: [pandion/pandion] New features and bug fixes (#10<#10>) Hi Maico, This is the screenshot of error [image]https://user-images.githubusercontent.com/48030433/144604381-1bee154d-2916-4a41-b86d-6e20d769e58a.png In this forum (https://soft.mydiv.net/win/qans.t52881.ne-rabotayut-smailiki-v-pandion.0-Pandion.html) they comment that the condition always gives false, which is why it is unnecessary to carry out this evaluation. When I comment lines 471 to 473 I get this result: [image]https://user-images.githubusercontent.com/48030433/144605659-f30a8bc6-0706-4fa1-ad41-707fb574d68f.png Thanks — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub<#10 (comment)<[#10 (comment)](https://github.com/pandion/pandion/pull/10#issuecomment-985494803)>>, or unsubscribehttps://github.com/notifications/unsubscribe-auth/ANZXPOCXWQLKPX3MTTVHJGDUPC4TLANCNFSM4J2BU52Q. Triage notifications on the go with GitHub Mobile for iOShttps://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675 or Androidhttps://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub. — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub<[#10 (comment)](https://github.com/pandion/pandion/pull/10#issuecomment-985524753)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ANZXPOHX2NN7ER7TA5U2FOLUPDBKTANCNFSM4J2BU52Q>. Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

Neustradamus commented:
@dries, @jinesh, @sebdeckers: I think that now it is possible to update the team no?
Ten years after?

Commit:
Versão 130

- Corrigido erro ao adicionar usuário
- Traduzidas novas funcionalidades para espanhol

AlexByte commented:
@maicotamanho why don't you use the releases section on github instead of google drive?

Neustradamus commented:
@AlexByte: We need to have an answer of @dries, @jinesh, @sebdeckers to have ownership in Pandion organization.

My goal, and the same of @maicotamanho is to relive the project.

I try since several years.

Badly the domain name has been lost.
I have spoken several years ago to pay it but the answer was "no".

AlexByte commented:
@Neustradamus i mean this repository https://github.com/maicotamanho/pandion

maicotamanho commented:
> why don't you use the releases section on github instead of google drive?

Hi.
Excuse me. I'm new to Github and didn't know this was possible.
Now is it ok?

AlexByte commented:
@maicotamanho 👍

GreenLunar commented:
> @AlexByte: We need to have an answer of @dries, @jinesh, @sebdeckers to have ownership in Pandion organization.

You don't need the handler @pandion nor github, there is gitlab, gitea. gogs and many more...
If you want to use github, then use @pandion-ng and continue from there.

GreenLunar commented:
@AlexByte Please open an issue tracker for bugs and feature requests

Neustradamus commented:
@GreenLunar: It is a solution to create a new name but it is very bad :/

I do not understand why @dries, @jinesh, @sebdeckers ignore since several years...

GreenLunar commented:
> @GreenLunar: It is a solution to create a new name but it is very bad :/

You're shraing code, and you're not forking in order to compete Pandion, you're forking to improve it.

> I do not understand why @dries, @jinesh, @sebdeckers ignore since several years...

Don't worry, they will join once the project is revived and then we'll negotiate.

I strongly suggest: **Do it**.

maicotamanho commented:
As soon as I started making corrections and improvements I contacted @sebdeckers. His return was this:

![ret](https://user-images.githubusercontent.com/57898936/158587340-7516a5ec-4655-4746-a72e-c03e4811e9b0.png)


I'm short on time to add improvements and adapt features to the new XMPP methods. I had and still have a lot of difficulty because I don't know the protocol. The company I work for is migrating to Teams. But Pandion proves to be very efficient, light and stable.

Neustradamus commented:
There are no progress since a very long time badly, for you and for me too.

There are to recover:
- GitHub: https://github.com/pandion
- SF: https://sourceforge.net/projects/pandion
- Google Group: http://groups.google.com/group/pandion
- Facebook: https://www.facebook.com/pandion.im
- Twitter: https://twitter.com/pandion
- OpenHub: https://www.openhub.net/p/pandion
- Flickr: https://www.flickr.com/photos/cbaspandion
- https://pandion.uptodown.com
- Others?

GreenLunar commented:
@Neustradamus please don't be sceptic, if you are (I'm getting that conclusion by your words)

@maicotamanho please take over the project and believe me the original developer will join you later.

Gentlemen, this looks like an unnecessary drama discussion.  Please stop this and continue the project.

GreenLunar commented:
Gentlemen, if no one will open a new organization (e.g. @pandion-ng) soon, then I'll do it myself and ask you to join it.

Neustradamus commented:
It is done!

GreenLunar commented:
Great. Fork the code and add new friends to the team. Don't add me please, I'll just follow.

Neustradamus commented:
@maicotamanho: Can you validate it?

And change your organization visibility to public?
- https://github.com/orgs/pandion-ng/people

GreenLunar commented:
@maicotamanho Please join pandion-ng and submit your PR there https://github.com/pandion-ng/pandion-ng

maicotamanho commented:
Hi, sorry for the delay, but I have a lot of work. 

> @maicotamanho Please join pandion-ng and submit your PR there https://github.com/pandion-ng/pandion-ng

I don't know how to do it.

Unfortunately the company I work for will be abandoning Pandion next month and I will no longer have the test environment.

Neustradamus commented:
@maicotamanho: I have relaunched, please look your email to validate it.

It is your ending?

Neustradamus commented:
@maicotamanho: Have you progressed?

sjehuda commented:
Gentlemen, I'd like to know what's the progress on this.
I want to add it to the list of recommended software for Windows platform.

Demo:
https://join.jabber.network/#movim@conference.movim.eu?join
Code:
https://github.com/ge0rg/easy-xmpp-invitation/blob/master/clients_Windows.json


Thank you

Neustradamus commented:
@maicotamanho: Have you looked for Pandion NG before changes from @sebdeckers?
